mod/beacon/validator/propose.go

```diff
@@ -298,3 +250,65 @@ func (s *Service[
 	}
 	return envelope, nil
 }
+
+// BuildBlockBody assembles the block body with necessary components.
+func (s *Service[
+	BeaconBlockT, BeaconBlockBodyT, BeaconStateT,
+	BlobSidecarsT, DepositStoreT, ForkDataT,
+]) buildBlockBody(
+	ctx context.Context,
+	st BeaconStateT,
+	blk BeaconBlockT,
+	reveal bytes.B96,
```

Comment:
common.BLSSignature

Reply:
@itsdevbear tks for reviewing. updated, is it`crypto.BLSSignature`?

Reply:
ya sorry